mozilla_schema_generator/schema.py

```diff
@@ -145,3 +145,67 @@ def _get_schema_size(schema: dict, key=None) -> int:
 
         # Otherwise, assume a scalar value
         return 1
+
+    @staticmethod
+    def get_type_as_list(t):
+        if isinstance(t, list):
+            return t
+
+        elif isinstance(t, str):
+            return([t])
+
+        elif isinstance(t, dict):
+            if "type" in t:
+                return(Schema.get_type_as_list(t["type"]))
+            elif "enum" in t:
+                return ["string"]
+
+        else:
+            raise Exception("Unknown JSON Schema `type` type " + str(type(t)))
+
+    @staticmethod
+    def get_invalid_schema_changes(_from, _to, verbosity=0):
```

Comment:
We should be able to use this to check that schema changes in MPS are compatible.